src/node/interfaces.cpp

```diff
@@ -100,8 +100,29 @@ class GOVImpl : public GOV
 public:
     void getAllNewerThan(std::vector<CGovernanceObject> &objs, int64_t nMoreThanTime) override
     {
-        if (governance == nullptr) return;
-        governance->GetAllNewerThan(objs, nMoreThanTime);
+        if (::governance != nullptr) {
+            return ::governance->GetAllNewerThan(objs, nMoreThanTime);
+        }
+        return;
+    }
+    int32_t getObjAbsYesCount(const CGovernanceObject& obj, vote_signal_enum_t vote_signal) override
+    {
+        // TODO: Move GetListAtChainTip query outside CGovernanceObject, query requires
+        //       active CDeterministicMNManager instance
+        if (::governance != nullptr && ::deterministicMNManager != nullptr) {
```

Comment:
nit: maybe better to invert condition?

```cpp
if (::governance == nullptr || deterministicMNManager == nullptr) return 0;
return obj.GetAbsoluteYesCount(vote_signal);
```

Reply:
Bitcoin has set the convention that you check for a positive determination and then act, as against, checking for a negative determination and fast-failing (personally, I'm a fan of the latter, it reduces so much indentation).

We can observe it in the `v0.21-final` branch ([link](https://github.com/bitcoin/bitcoin/blob/v0.21-final/src/interfaces/node.cpp#L119-L156)) as well as on the `23.x` branch, the oldest supported release ([link](https://github.com/bitcoin/bitcoin/blob/23.x/src/node/interfaces.cpp#L121-L185)). In accordance to that, they were formatted.

Though it is to be noted that `ChainImpl` seems to do the opposite (fast-failing instead of testing positive conditions) but I assume we're considering Dash extensions to be children of `NodeImpl`.